Telegram/SourceFiles/info/profile/info_profile_music_button.cpp

```diff
@@ -83,11 +83,11 @@ int MusicButton::resizeGetHeight(int newWidth) {
 	const auto skip = st::normalFont->spacew;
 	const auto available = newWidth - padding.left() - padding.right();
 	_title->resizeToNaturalWidth(available);
-	_title->moveToLeft(padding.left(), top);
-	if (const auto left = available - _title->width() - skip; left > 0) {
+	_title->moveToLeft(padding.left() + _performer->width() + skip, top);
+	if (const auto left = available - _performer->width() - skip; left > 0) {
```

Comment:
The problem 23rd shows is due to this change. Revert it back to
```suggestion
	if (const auto left = available - _title->width() - skip; left > 0) {
```